libs/langchain/langchain/chains/combine_documents/stuff.py

```diff
@@ -91,9 +96,16 @@ def format_docs(inputs: dict) -> str:
     ).with_config(run_name="stuff_documents_chain")
 
 
+@deprecated(
```

Comment:
@xdumaine we can merge the rest of this except for the deprecation. We're trying to figure out how to deprecate some of these things without causing too much churn for users that have existing code that's relying on this. 

cc @ccurme  -- has the most knowledge here in terms of which chains are safe to deprecate

Reply:
@eyurtsev no problem, thank you for the review/feedback!